src/main/java/uk/ac/ebi/ampt2d/accession/server/FileAccessionResponse.java

```diff
@@ -28,6 +28,9 @@
 
     private List<FileAccession> accessions;
 
+    public FileAccessionResponse() {
```

Comment:
Not Needed unless it is gonna be used for a post parsing, and in that case package protected